### PR TITLE
Unit test for Devops Tooling dashboard

### PR DESCRIPTION
**Pull request by dhiraj-wakchaure**
wrote
unit test for components
snapshot tests
with shallow and deep rendering

### CODE REVIEW

File: devops-tooling/dashboard/src/components/Dashboard/Dashboard.test.tsx [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
+    const dashboardElement = screen.getByTestId('dashboard');
+    expect(dashboardElement).toBeInTheDocument();
   });
-
-  test('It should mount', () => {
-    expect(component.length).toBe(1)
-  })
 })
+
+// describe('deep rendering <Dashboard />', () => {
```
Comment (JaneMcBrain) [inline devops-tooling/dashboard/src/components/Dashboard/Dashboard.test.tsx:18]: do we still need this?

File: devops-tooling/dashboard/src/components/Link/Link.test.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { render } from "@testing-library/react";
+import Link from "./Link";
+
+
```
Comment (JaneMcBrain) [inline devops-tooling/dashboard/src/components/Link/Link.test.tsx:4]: can we remove empty lines? and the indentation looks weird in my IDE

**Review by JaneMcBrain — CHANGES_REQUESTED** [review]

File: devops-tooling/dashboard/src/components/Dashboard/Dashboard.test.tsx (hunk 1 above)
Comment (dhiraj-wakchaure) [reply devops-tooling/dashboard/src/components/Dashboard/Dashboard.test.tsx:18]: I kept that as example first , agree that can remove as covered in others

Comment (dhiraj-wakchaure) [reply devops-tooling/dashboard/src/components/Dashboard/Dashboard.test.tsx]: removed

**Review by JaneMcBrain — APPROVED** [review]

**Pull request merged**